spec/features/users/verify_profile_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+feature 'verify profile with OTP' do
+  let(:user) { create(:user, :signed_up) }
+  let(:otp) { 'abc123' }
+
+  before do
+    create(
+      :profile,
+      deactivation_reason: :verification_pending,
+      pii: { otp: otp, ssn: '666-66-1234', dob: '1920-01-01' },
+      user: user
+    )
+  end
+
+  scenario 'received OTP via USPS' do
+    sign_in_live_with_2fa(user)
+
+    expect(current_path).to eq profile_path
+
+    click_on t('profile.index.verification.reactivate_button')
+
+    expect(current_path).to eq verify_profile_path
+
+    fill_in 'Secret code', with: otp
+    click_button t('forms.verify_profile.submit')
+
+    expect(current_path).to eq profile_path
+    expect(page).to_not have_content(t('profile.index.verification.reactivate_button'))
+  end
+
+  xscenario 'OTP has expired' do
```

Comment:
We've had about the same set of pending tests in this app since I started working on it ~6 months ago, so I automatically ignore them. If the pending test is a prompt for you, that's cool.